scripts/write_complete_data.py

```diff
@@ -38,16 +43,15 @@ def _modify_relative_data(relative_data):
             string = RELATIVE_PATTERN.sub(r'(\\d+)', string)
             value[i] = string
         modified_relative_data[key] = value
-    return modified_relative_data
```

Comment:
If this is not going to return the `modified_relative_data` dict we don't even need to create it because we don't need to track it.

suggestion: deleting `modified_relative_data = OrderedDict()` and `modified_relative_data[key] = value`.